netmiko/endace/endace_ssh.py

```diff
@@ -1,18 +1,36 @@
+from typing import Any
+
 from netmiko.cisco_base_connection import CiscoSSHConnection
 import re
 
 
 class EndaceSSH(CiscoSSHConnection):
-    def disable_paging(self, command="no cli session paging enable", delay_factor=1):
+    def disable_paging(
+        self,
+        command: str = "no cli session paging enable",
+        delay_factor: float = 1.0,
+        *args: Any,
+        **kwargs: Any
+    ) -> str:

```

Comment:
You should pass *args and **kwargs in the super call so that arguments are not dropped.